kolaintro.md

```diff
@@ -0,0 +1,54 @@
+ # Kola Introduction Documentation
+ 
+ This document just goes through the basics of how kola test's work and how someone just beginning can get started
+ 
+ 
+ ### Useful Link
+ 
+You can get more information about it, in the oficial documentation:
+https://github.com/coreos/coreos-assembler/blob/main/docs/kola/external-tests.md
+
+ 
+ #### Steps to run kola
+
+
+Here we can see how to run kola: https://gitlab.cee.redhat.com/coreos/team-operations/-/blob/b96d3032611896d905f33907e5d2258706a1b5e1/PIPELINES.md#debugging
```

Comment:
Let's not link to internal repositories here. If you think that information is useful for any user, I'd suggest moving that content out of the internal repo and into a documentation here.